.changeset/tidy-comics-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@voucherify/sdk': minor
+---
+
+#### Update axios from `0.21.4` to `0.27.2` in SDK package
```

Comment:
1. This update does not follow the template (header used)
2. This update does not contain information about: a) added template b) added frame for integration tests c) added dotenv to dependencies

Reply:
`added changelog template` - nope - template is not part of packages/sdk folder. I'll add a note in root readme about it.
You are correct about everything else.